.travis.yml

```diff
@@ -11,6 +11,10 @@ install:
   - docker-compose run web bash -c "rake db:setup"
   - docker-compose run web bash -c "rake db:migrate"
   - docker-compose run web bower install --allow-root
+  - 'if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then gem install danger danger-junit; fi'
 
 script:
-  - docker-compose run web bash -c "rake test"
+  - docker-compose run web bash -c "CI=TRUE GENERATE_REPORT=true rake test"
```

Comment:
Hmm, oddly `plots2` uses `exec` instead of `run` here. But that didn't work. So trying `run` as it was originally here.

Reply:
`ERROR: No container found for web_1` was the error...